src/helm/benchmark/scenarios/opinions_qa_scenario.py

```diff
@@ -0,0 +1,172 @@
+import os
+import pandas as pd
+from typing import List, Dict
+
+from helm.common.general import ensure_file_downloaded
+
+from .scenario import (
+    Scenario,
+    Instance,
+    Reference,
+    TRAIN_SPLIT,
+    TEST_SPLIT,
+    CORRECT_TAG,
+    PassageQuestionInput,
+    Input,
+    Output,
+)
+
+
+class OpinionsQAScenario(Scenario):
+    """
+    The OpinionsQAScenario dataset is from the paper "Whose Opinions Do Language Models Reflect?"
+    [Santurkar et al., 2023].
+
+    OpinionsQA is a QA dataset containing 1484 multiple-choice questions. Since the questions are inherently
+    subjective, there isn't a single ground truth response. Instead, the object of interest is how
+    the distribution of model responses compares to those obtained from human survey participants.
+
+    As discussed in Santurkar et al., we consider prompting an LM:
+    1. Without any context (zero-shot) to evaluate the "default" opinions reflected
+        by it.
+    2. With context containing information pertaining to the group we want to steer the model towards.
+       This context is either formatted as a question-answer pair (QA) or a textual description (BIO/PORTRAY).
```

Comment:
Could we specify precisely the values of `context`?  `"default"`, `"steer-qa"`, etc.